protocol/block.rei

```diff
@@ -19,9 +19,16 @@ type t =
 let sign: (~key: Address.key, t) => Signature.t;
 let verify: (~signature: Signature.t, t) => bool;
 let genesis: t;
+
+type next_hashes = {
+  state_root: BLAKE2B.t,
+  validators: BLAKE2B.t,
+};
```

Comment:
I believe next_hashes is not needed anymore after #246 and it can be replaced by next_state_root_hash parameter on produce

Reply:
fixed in latest commit